apps/omg_watcher/lib/block_getter/o_m_g/watcher/block_getter/supervisor.ex

```diff
@@ -0,0 +1,42 @@
+# Copyright 2018 OmiseGO Pte Ltd
```

Comment:
sth's wrong with the path, should be `lib/block_getter/supervisor.ex` (`OMG.Watcher` part is "consumed" by `apps/omg_watcher` according to our conventions

Reply:
sorry, it it caused by the fact that I did create this file via module creator and did not realize that it was created in a strange directory